src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java

```diff
@@ -11,46 +14,73 @@
 public class Command_gcmd extends FreedomCommand
 {
 
+    public static final List<String> BLOCKED_COMMANDS = Arrays.asList(
+            "wildcard",
+            "gtfo",
+            "doom",
+            "saconfig",
+            "gcmd",
+            "glist",
+            "tban",
+            "tempban"
+    );
+
     @Override
     public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
     {
-        if (args.length < 2)
+        try
         {
-            return false;
-        }
+            final Player player = getPlayer(args[0]);
 
-        final Player player = getPlayer(args[0]);
+            if (args.length < 2)
+            {
+                return false;
+            }
 
-        if (player == null)
-        {
-            sender.sendMessage(FreedomCommand.PLAYER_NOT_FOUND);
-            return true;
-        }
+            if (player == null)
+            {
+                sender.sendMessage(FreedomCommand.PLAYER_NOT_FOUND);
+                return true;
+            }
 
-        final String outCommand = StringUtils.join(args, " ", 1, args.length);
+            String outCommand = StringUtils.join(args, " ", 1, args.length);
+            Command runCmd = server.getPluginCommand(args[1]);
+            
+            if (BLOCKED_COMMANDS.contains(runCmd.getName()))
+            {
+                msg("Blocked. Yes, I mean blocked.", ChatColor.RED);
+                return true;
+            }
 
-        if (plugin.cb.isCommandBlocked(outCommand, sender))
-        {
-            return true;
-        }
+            if (plugin.cb.isCommandBlocked(outCommand, sender))
+            {
+                return true;
+            }
 
-        try
-        {
-            msg("Sending command as " + player.getName() + ": " + outCommand);
-            if (server.dispatchCommand(player, outCommand))
+            try
             {
-                msg("Command sent.");
+                msg("Sending command as " + player.getName() + ": " + outCommand);
+                if (server.dispatchCommand(player, outCommand))
+                {
+                    msg("Command sent.");
+                    player.sendMessage(sender.getName() + " just ran the command: [/" + outCommand + "]" + " as you!");
+                    FLog.info("Alert! User: " + sender.getName() + " just ran the command: [/" + outCommand + "]" + " as " + player.getName());
```

Comment:
This would be purely info, not an alert, please loose the Alert! part at the start.